src/releng_docs/projects/shipit-code-coverage-crawler.rst

```diff
@@ -0,0 +1,16 @@
+.. shipit-code-coverage-crawler-project:
+
+Project: shipit-code-coverage-crawler
+=====================================
+
+:contact: `Marco Castelluccio`_, (backup `Release Management`_)
+
+This project uses the `Code Coverage Crawler`_ to navigate pages from a list of
+websites and publishes reports of code that is being run by navigating those
+pages, but not by our tests. It's using `firefox-code-coverage`_ to generate
```

Comment:
`firefox-code-coverage` is used to download Fiferox full coverage report, that crawler later differs with the report generated while running websites; and, finally, It uses `genhtml` to generate HTML code coverage reports from those differences reports.

Reply:
Thanks for the info, I updated this.